.github/workflows/qaqc.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
     # NOTE: this isn't a comprehensive spellcheck, just common typos
     - name: Spellcheck
       if: always()
-      uses: codespell-project/actions-codespell@master
+      uses: codespell-project/actions-codespell@v2
```

Comment:
As a side note and personal practice - I am fan of splitting such commits into two. Makes it easier to transfer this to the main template without causing any conficts.

Reply:
multi-tasking and rushing :)